src/util/sfdc.ts

```diff
@@ -127,8 +127,19 @@ export const sfdc = {
    * @param url
    */
   isInternalUrl: (url: string): boolean => {
-    const INTERNAL_URL_PARTS = ['.internal.', '.vpod.', 'stm.salesforce.com', '.blitz.salesforce.com'];
+    const INTERNAL_URL_PARTS = ['.vpod.', 'stm.salesforce.com', 'stm.force.com', '.blitz.salesforce.com'];
+    return (
+      url.startsWith('https://gs1.') || sfdc.isLocalUrl(url) || INTERNAL_URL_PARTS.some((part) => url.includes(part))
```

Comment:
Does anyone remember the context of `gs1` check? It feels so out of place to me.

Reply:
kept it because it was here: https://github.com/forcedotcom/sfdx-core/blob/10fc422fe87b2ba2fb940abd07d0742f78f044f3/src/authInfo.ts#L231

for jwt audience stuff